lib/player.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'scoring'
+require_relative 'tile_bag'
+module Scrabble
+  class Player
+    attr_accessor :name, :plays, :total_score, :tiles
+    def initialize(name)
+      @name = name
+      @plays = []
+      @total_score = 0
+      @tiles = []
+    end
+
+    def play(word)
+      if won?
+        return false
+
+      else
+        word_check = []
+        word_chars = word.upcase.chars
+        word_chars.each do |char|
+          if !tiles.include?(char.to_sym)
+            return false
+          else
+            word_check << char.to_sym
+          end
+        end
+
+        plays << word
+        score = Scrabble::Scoring.score(word)
+        self.total_score += score
```

Comment:
This totally works, but you might consider making `total_score` a calculated field (a method) instead of an instance variable.